client/src/assets/scss/base.scss

```diff
@@ -9,3 +9,11 @@
 /* HTML element styles should go here */
 
 /* global styles should go here */
+
+
+.pr-75 {
+  padding-right: 75px;
+}
+.pl-75 {
```

Comment:
I see what you're trying to do and I think it's fine, but I think you should add `!important` to both of these so ensure they override any other padding specs (just as the Vuetify spacing helper classes do).